### PR TITLE
68 seo helmet setup

### PR DESCRIPTION
**Pull request by ngrujicic**

### CODE REVIEW

File: src/app/router/routes.ts [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
 const HOME_ROUTE = "/";
 const POOLS_ROUTE = "pools";
 const ADD_LIQUIDITY = "add-liquidity";
 const SWAP_ROUTE = "swap";
 const POOLS_ADD_LIQUIDITY = "/pools/add-liquidity";
 const POOLS_PAGE = "/pools";
 
-export { HOME_ROUTE, POOLS_ROUTE, SWAP_ROUTE, ADD_LIQUIDITY, POOLS_ADD_LIQUIDITY, POOLS_PAGE };
+const SEO_ROUTES = {
+  [`${POOLS_ROUTE}`]: {
```
Comment (npanajotov) [inline src/app/router/routes.ts]: You can put just [POOLS_ROUTE], without `.

File: src/components/organism/SwapTokens/index.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                  <button
+                    className={`flex basis-1/2 justify-center rounded-lg  px-4 py-3 ${
+                      slippageAuto ? "bg-purple-100" : "bg-white"
+                    }`}
+                    onClick={() => setSlippageAuto(true)}
+                  >
+                    {t("tokenAmountInput.auto")}
+                  </button>
+                  <button
+                    className={`flex basis-1/2 justify-center rounded-lg px-4 py-3 ${
```
Comment (npanajotov) [inline src/components/organism/SwapTokens/index.tsx:107]: Please use classnames instead of ternary operator.

Comment (ngrujicic) [reply src/components/organism/SwapTokens/index.tsx:107]: done

File: src/app/router/routes.ts (hunk 1 above)
Comment (ngrujicic) [reply src/app/router/routes.ts]: it breaks now that i removed this, i am fixing it now

**Review by npanajotov — APPROVED** [review]

**Review by npanajotov — APPROVED** [review]

**Review by markoMVP85 — APPROVED** [review]

**Pull request merged**